### ISSUE TITLE
Search not working

### ISSUE THREAD

**Issue by lweiler77** [alexbevi/redmine_knowledgebase#330]
After installing the plugin, the search is no longer working.

> Internal errorAn error occurred on the page you were trying to access.

Is there a fix for this?

Cheers 

Environment:
  Redmine version                3.1.3.stable
  Ruby version                   2.1.7-p400 (2015-08-18) [x86_64-linux-gnu]
  Rails version                  4.2.5
  Environment                    production
  Database adapter               Mysql2
SCM:
  Subversion                     1.8.8
  Mercurial                      2.8.2
  Cvs                            1.12.13
  Bazaar                         2.7.0
  Git                            2.6.4
  Filesystem                     
Redmine plugins:
  redmine_knowledgebase          3.2.0

**Comment by shawndibble** [comment]
I am getting the same issue. It appears that redmine_knowledgebase causes a 500 error if I search for anything other than an issue id.

```
Completed 500 Internal Server Error in 44ms (ActiveRecord: 16.3ms)

NoMethodError (undefined method `where' for #<Hash:0x007f183961cd50>):
  lib/plugins/acts_as_searchable/lib/acts_as_searchable.rb:197:in `search_scope'
  lib/plugins/acts_as_searchable/lib/acts_as_searchable.rb:92:in `search_result_ranks_and_ids'
  lib/redmine/search.rb:127:in `block in load_result_ids'
  lib/redmine/search.rb:125:in `each'
  lib/redmine/search.rb:125:in `load_result_ids'
  lib/redmine/search.rb:115:in `block in load_result_ids_from_cache'
  lib/redmine/search.rb:114:in `load_result_ids_from_cache'
  lib/redmine/search.rb:99:in `result_ids'
  lib/redmine/search.rb:70:in `result_count'
  plugins/redmine_xapian/lib/redmine_xapian/search_controller_patch.rb:135:in `index_with_xapian'
  lib/redmine/sudo_mode.rb:63:in `sudo_mode'
  plugins/redmine_dmsf/lib/redmine_dmsf/webdav/no_parse.rb:33:in `call'

```

**Comment by kdadelit** [comment]
I'd got this trouble too on Redmine 3.3.0.

In _plugins/redmine_knowledgebase/app/models/kb_article.rb_ on line 42 I did removed "#{table_name}." substrings from columns array and all became good.

I didn't tested all functionality yes, but I think that this will be enough to have fun :-)

**Comment by alexbevi** [comment]
Thanks to @jdkemme this should now be resolved in v3.2.1
